### PR TITLE
Port Banner from Gatsby to Next [#1616]

### LINKED ISSUE #1616: [next] Port Banner from Gatsby to Next
**What would you like to be added**:
The banner should be ported from Gatsby to Next.

**Why would you like this to be added**:
We should display our banner with pride.

### PR DESCRIPTION
**Pull request by foxviet**
## Issue This PR Addresses

Fixes #1616 

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

It works, I would need to get back to it later.

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by PedroFonsecaDEV** [comment]
@foxviet It seems that you are dealing with some git issues here. 

cc @humphd

**Comment by chrispinkney** [comment]
I think you commited the `.cache` and the `public` folder, i'm not sure but maybe try `git rm`ing them and pushing, might solve the issue.

**Review by humphd — CHANGES_REQUESTED** [review]
Yes, please remove `src/frontend/.cache` from this

File: src/frontend/next/out/404.html [hunk 1]
```diff
@@ -0,0 +1 @@
...lign:middle"><h2 style="font-size:14px;font-weight:normal;line-height:inherit;margin:0;padding:0">This page could not be found<!-- -->.</h2></div></div></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"statusCode":404}},"page":"/_error","query":{},"buildId":"A9UYMyJSJ3cx9OK58xeoM","nextExport":true,"isFallback":false,"gip":true}</script><script nomodule="" src="/_next/static/chunks/polyfills-4acaa4b69e5977b3ab87.js"></script><script src="/_next/static/chunks/main-0db40ba1b1ffc82919b2.js" async=""></script><script src="/_next/static/chunks/webpack-50bee04d1dc61f8adf5b.js" async=""></script><script src="/_next/static/chunks/framework.a4b3ed6c46c358b601ab.js" async=""></script><script src="/_next/static/chunks/f6078781a05fe1bcb0902d23dbbb2662c8d200b3.d4f57090bd55662b89aa.js" async=""></script><script src="/_next/static/chunks/pages/_app-bde721f06a5022f001d6.js" async=""></script><script src="/_next/static/chunks/pages/_error-e2ffa4674eea62b50f3f.js" async=""></script><script src="/_next/static/A9UYMyJSJ3cx9OK58xeoM/_buildManifest.js" async=""></script><script src="/_next/static/A9UYMyJSJ3cx9OK58xeoM/_ssgManifest.js" async=""></script></body></html>
```
Comment (chrispinkney) [inline src/frontend/next/out/404.html:1]: Can you remove the `out` folder too? `git add .` is generally not your friend 😋

**Review by chrispinkney — APPROVED** [review]

File: src/frontend/next/out/404.html (hunk 1 above)
Comment (foxviet) [reply src/frontend/next/out/404.html:1]: I removed out/cache folders, commited it but it just created another branch, not to this PR 😀

**Review by HyperTHD — CHANGES_REQUESTED** [review]
You should revert this PR to commit 5c1fad94f5ba76fc1e725ba62f1d4da572c12f57 so that you can remove all those files that you included by accident with the merge. Once you do that, fix up the Banner component, and then force-push. You still got this

**Comment by foxviet** [comment]
> You should revert this PR to commit [5c1fad9](https://github.com/Seneca-CDOT/telescope/commit/5c1fad94f5ba76fc1e725ba62f1d4da572c12f57) so that you can remove all those files that you included by accident with the merge. Once you do that, fix up the Banner component, and then force-push. You still got this

@HyperTHD 
#5c1fad9 The 'next' folder seemed 'empty' to me

**Comment by humphd** [comment]
The DynamicImage component is up at #1634, which this will need.

**Pull request closed without merging**